exec/call_test.go

```diff
@@ -99,11 +99,11 @@ var moduleCallHost = []byte{
 	0x08, 0x01, 0x06, 0x00, 0x41, 0x00, 0x10, 0x00, 0x0B,
 }
 
-func add3(x int32) int32 {
+func add3(vi *Process, x int32) int32 {
 	return x + 3
 }
 
-func importer(name string) (*wasm.Module, error) {
+func importer(name string, f func(*Process, int32) int32) (*wasm.Module, error) {
```

Comment:
we could modify `importer` to take the `f func(*Process, int32) int32)` argument, close around it and return the usual `func(string) (*wasm.Module, error)` importer function...

(just a suggestion. it might make things less obvious, though...)

Reply:
I thought of that, and I think this is not the right approach: I can only imagine one real-life use case when a host function needs not access the memory: reading integer value from a system-specific file (like `/proc/$$/fd` or `/dev/urandom`) and that is by no means the most general use case, so it feels like an unnecessary complexity to me. 
Furthermore, the spec specifies that the store should be passed to host functions so it seems normal to do that in any case, just as the spec expects an interpreter to.

Reply:
ok.